src/lexer/Tokenizer.ts

```diff
@@ -152,7 +152,7 @@ export default class Tokenizer {
       },
       {
         type: TokenType.OPERATOR,
-        regex: regex.operator('+-/%&|^><=.:$@#?~!', [
+        regex: regex.operator('-+/%&|^><=:$@#?~!', [
```

Comment:
Discovered a grave error in this code.

Previously this string started with `+-/`, which ended up being placed inside regex character class, like `[+-/]`. Which instead of matching `+`, `-` and `/`, ended up matching all characters between `+` and `/` (which by chance happens to include `+`, `,`, `-`, `.`, `/`).